src/components/common/AddEditOrgPerson.vue

```diff
@@ -114,55 +114,125 @@
 
             <!-- Roles -->
             <article class="mt-2">
-              <div class="font-weight-bold">Roles</div>
-              <v-card flat rounded="sm" class="gray-card mt-4 px-4">
-                <v-row class="align-center">
-                  <v-col cols="4" v-if="showCompletingPartyRole" class="py-0">
-                    <v-checkbox
-                      id="cp-checkbox"
-                      class="mt-5"
-                      v-model="selectedRoles"
-                      :value="RoleTypes.COMPLETING_PARTY"
-                      :label="RoleTypes.COMPLETING_PARTY"
-                      :disabled="disableCompletingPartyRole"
-                      @change="assignCompletingPartyRole()"
-                    />
-                  </v-col>
+              <div class="font-weight-bold">{{ getPeopleAndRolesResource.rolesTitle || 'Roles' }}</div>
+              <div v-if="getPeopleAndRolesResource.rolesSubtitle">
+                {{ getPeopleAndRolesResource.rolesSubtitle }}
+              </div>
+              <div class="form__row three-column mt-4">
+                <v-card flat rounded="sm" class="item gray-card px-4">
+                  <v-row no-gutters class="align-center mt-4">
+
+                    <v-col cols="4" v-if="showCompletingPartyRole">
```

Comment:
I updated the layout to support less than, or more than, 3 items per line. Basically they are all "3-wide columns" within a single row, so they wrap.

Before:
![image](https://user-images.githubusercontent.com/10002889/212783120-fdc9c3ed-57e7-4dbe-b0e0-1a59391acd30.png)

After (IA):
![image](https://user-images.githubusercontent.com/10002889/212783154-7c587337-5355-4688-8634-50b8bc8256b4.png)

After (dissolution):
![image](https://user-images.githubusercontent.com/10002889/212783175-c78272a6-8863-4f72-9526-83db163ffd2e.png)